integrations/opennms-vmware/src/main/java/org/opennms/netmgt/provision/service/vmware/VmwareRequisitionUrlConnection.java

```diff
@@ -130,11 +130,22 @@ public VmwareRequisitionUrlConnection(URL url) throws MalformedURLException, Rem
 
         m_hostname = url.getHost();
 
-        m_username = getUsername();
-        m_password = getPassword();
+        logger.debug("Initializing URL Connection for host {}", m_hostname);
 
         m_args = getQueryArgs();
 
+        // Old or new user credentials handling scheme
+	if (url.getUserInfo() != null && !url.getUserInfo().isEmpty()) {
+            logger.warn("Old user credentials handling scheme detected. I'm gonna use it but you'd better adapt your URL to the new query parameter scheme 'vmware://<vcenter_server_fqdn>?username=<username>;password=<password>;....'");
+            m_username = getUsername();
+            m_password = getPassword();
+        } else {
+            m_username = m_args.get("username");
+            m_password = m_args.get("password");
+        }
```

Comment:
Well, there's already one: 

```
Logger.warn("Old user credentials handling scheme detected. I'm gonna use it but you'd better adapt your URL to the new query parameter scheme 'vmware://<vcenter_server_fqdn>?username=<username>;password=<password>;....'");
```

It will be printed whenever the old user credentials handling scheme is detected. So, also when both the old and the new schemes are specified.

I personaly don't see the necessity to add another test and log 

Cyrille